Dockerfiles/ccp-openshift-scan/scanning/yumupdates.py

```diff
@@ -1,44 +1,117 @@
-# scan script for yum list update scannr
+# scan script for yum list update scanner
 
-import scan_lib
+import sys
+import yum
 
 
-def yum_updates():
+class SysStdoutSuppressor(object):
```

Comment:
When we were doing yum updates checking we're providing `-q` option to be quiet for yum repo updates download stdout. This is same implementation in python. We don't want to print anything other than the updates available, helps the logs fetching to do less processing.

Reply:
If we don't do this, the python processing for finding updates will print on stdout.

Reply:
Basically, without this, its `yum check-update`

Reply:
https://www.python.org/dev/peps/pep-0343/ this should help review this.